app/test_main.py

```diff
@@ -1,3 +1,27 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, human_age",
+    [
+        (0, 0, [0, 0]),
+        (-10, -10, [0, 0]),
+        (100000, 100000, [24996, 19997]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 29, [3, 3]),
+    ]
+)
```

Comment:
Looks good, but add more test data to check all age boundaries, for example, that 23 returns 1 or 27, 28 returns 2

Reply:
Done